rust/private/rustc.bzl

```diff
@@ -460,7 +460,9 @@ def construct_arguments(
         build_flags_files,
         emit = ["dep-info", "link"],
         force_all_deps_direct = False,
-        stamp = False):
+        force_link = False,
```

Comment:
Would it be possible to add a unit test demonstrating the usage of this parameter and its expected effect (probably on the generated rustc command line?) Same for `remap_path_prefix`.

Reply:
Sorry about the delay on this one: https://github.com/bazelbuild/rules_rust/pull/1078